design/src/high-level/creative-automation-case-studies.md

```diff
@@ -0,0 +1,282 @@
+# Creative Automation: Case Studies
+
+## Emergence
+
+**Conclusion:** Creativity will be low until more systems are in place, but potential is excellent. Clear and satisfying aesthetics will be key to creating a compelling contraption. Disruptions should be tunable to ensure every player can reach an optional presure level.  
+
+### Interesting Goals
+
+Standard factory building goals are present,
+but an emphasis on sustainability and resilience add even more goals to consider.
+
+Stronger spatial and temporal variability increase the importance of considering tradeoffs and tailoring solutions to conditions.
+
+### Distinct Paths
+
+Initial prototypes will be quite weak, simply due to scale.
+
+Adding more systems and organisms will enhance this substantially.
+
+### Low Pressure
+
+Aesthetics are well-suited to chill alien cottage-core vibes.
+Disruptions are planned as an important gameplay mechanic: need excellent tuning levers to ensure that space for creativity isn't removed.
+Cascading failures shouldn't be too severe by default.
+
+### Compelling Aesthetics
+
+Smashing success.
+Unique fantasy that lends itself well to a bustling, automated hub.
+
+Aesthetic differentation will require more content than MVP.
+
+Potential for very compelling contraptions.
+Audio and timing and visual clarity will be tricky but essential to creating good contraptions.
+
+## [Factorio](https://store.steampowered.com/app/427520/Factorio/)
+
+**Conclusion:** top-tier creative automation, especially in modded games.
+Lots of options that differ in interesting ways.
+Limited aesthetic choices.
+
+### Interesting Goals
+
+Science construction is interesting and clear.
+Secondary optimization objectives are often meaningful.
+
+Trains and bots are great end products: powerful and intrinsically satisfying.
+Player upgrades like cars and suits serve as additional chase goals.
+While combat is frustrating, it serves as a clear motivator in games where it's enabled.
```

Comment:
Why do you think combat in Factorio is frustrating?

I think possible reasons are:
- Provides no direct reward other than protecting the Factory
- While there are different units, there is no real variance, every battle feels more of less the same

Reply:
I agree with both of those reasons.

I also think it's because:

3. It feels unjustified: you're just being a dick and wrecking stuff.
4. There's no sense of progress when biter expansion is turned on.
5. The actual mechanics feel bad: it's simply not a good combat game due to tuning and game feel. Worms are particularly irritating, spitters ignore walls, killing nests feels like a bullet sponge.
6.  Losing non-renewable resources into the void to fight off a never-ending wave feels miserable (even if it doesn't matter).
7. There's a risk (perceived or real) of snowballing destruction: players will often save scum rather than accept any losses.

Most of these are just "make combat good instead", rather than fundamental problems of mixing it with the genre IMO.